zeroex/order.go

```diff
@@ -351,17 +352,17 @@ func (s SignedOrder) MarshalJSON() ([]byte, error) {
 	}
 
 	signedOrderBytes, err := json.Marshal(SignedOrderJSON{
-		MakerAddress:          s.MakerAddress.Hex(),
+		MakerAddress:          strings.ToLower(s.MakerAddress.String()),
```

Comment:
```suggestion
		MakerAddress:          strings.ToLower(s.MakerAddress.Hex()),
```

Nit: `String()` simply points to `Hex()` under-the-hood and exists as an implementation of the `stringer` interface. I feel it's slightly more expressive to use `Hex()` here. Mind making this change in the other places too?

Reply:
Of course, I totally agree. I tried `.String()` (before checking implementation) to see if go-ethereum offered a native non-checksummed stringer, and never reverted.